google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolTest.java

```diff
@@ -16,12 +16,7 @@
 
 package com.google.cloud.spanner;
 
-import static com.google.cloud.spanner.MetricRegistryConstants.NUM_IN_USE_SESSIONS;
-import static com.google.cloud.spanner.MetricRegistryConstants.NUM_READ_SESSIONS;
-import static com.google.cloud.spanner.MetricRegistryConstants.NUM_SESSIONS_BEING_PREPARED;
-import static com.google.cloud.spanner.MetricRegistryConstants.NUM_WRITE_SESSIONS;
-import static com.google.cloud.spanner.MetricRegistryConstants.SPANNER_LABEL_KEYS;
-import static com.google.cloud.spanner.MetricRegistryConstants.SPANNER_LABEL_KEYS_WITH_TYPE;
+import static com.google.cloud.spanner.MetricRegistryConstants.*;
```

Comment:
```suggestion
import static com.google.cloud.spanner.MetricRegistryConstants.NUM_IN_USE_SESSIONS;
import static com.google.cloud.spanner.MetricRegistryConstants.NUM_READ_SESSIONS;
import static com.google.cloud.spanner.MetricRegistryConstants.NUM_SESSIONS_BEING_PREPARED;
import static com.google.cloud.spanner.MetricRegistryConstants.NUM_WRITE_SESSIONS;
import static com.google.cloud.spanner.MetricRegistryConstants.SPANNER_LABEL_KEYS;
import static com.google.cloud.spanner.MetricRegistryConstants.SPANNER_LABEL_KEYS_WITH_TYPE;
```
Google style requires us to never use wildcard imports: https://google.github.io/styleguide/javaguide.html